app/models/git/has_git_filepaths.rb

```diff
@@ -0,0 +1,24 @@
+module Git
+  module HasGitFilepaths
+    def git_filepaths(**fields_to_filepaths)
+      fields_to_filepaths = Hash(fields_to_filepaths)
```

Comment:
I don't think we don't need this line.

Reply:
We probably don't. I just copy-pasted it.